public/pages/DetectorResults/containers/AnomalyHistory.tsx

```diff
@@ -416,6 +424,13 @@ export const AnomalyHistory = (props: AnomalyHistoryProps) => {
     : atomicAnomalyResults;
   const handleDateRangeChange = useCallback(
     (startDate: number, endDate: number, dateRangeOption?: string) => {
+      if (
+        !props.isHistorical &&
+        startDate < get(props, 'detector.enabledTime') &&
```

Comment:
Yeah, I had brought up the same concerns with UX initially about this. It was agreed that it is ok to show the modal even if there is results before the re-enabled time, for 2 reasons:
2. If the detector was stopped, had configuration changes, then re-enabled, it wouldn't make sense to consider the previous anomaly results as representative of the new detector config. In this case, we would want the modal to show

Reply:
I think it's a good idea - I'll sync with UX about this.

Reply:
It's agreed that the last enabled time should be present on the page; UX + tech writers are working on finalizing the wording but it will be added. Thanks for the suggestion @kaituo!